tasks/engine_setup.yml

```diff
@@ -0,0 +1,61 @@
+---
+- block:
+  - name: Use the default answerfile
+    template:
+      src: answerfile_{{ ovirt_engine_setup_version }}_basic.txt.j2
+      dest: /tmp/answerfile.txt
+      mode: 0600
+      owner: root
+      group: root
+    when: ovirt_engine_setup_answer_file_path is undefined
+
+  - name: Copy custom answer file
+    no_log: "True"
+    template:
+      src: "{{ ovirt_engine_setup_answer_file_path }}"
+      dest: /tmp/answerfile.txt
+      mode: 0600
+      owner: root
+      group: root
+    when: ovirt_engine_setup_answer_file_path is defined
+
+  - name: Update setup packages
+    shell: 'yum -y update ovirt*setup*'
```

Comment:
Is there a reason to not use yum module?

Reply:
yum module doesn't never run yum update, even if you specify latest. I always runs yum install + params.
When I used the ansible yum module, I was hitting dependency issues, when updating setup packages, and it was complaining about missing dependecies that I could they are in repository.
I can rerun with this change and check again, however. Maybe there was other issue.

Reply:
Fixed.